nexus_keycode/protocols/full.py

```diff
@@ -421,3 +425,14 @@ def passthrough_command(cls, application_id, passthrough_digits):
             raise ValueError("Passthrough body cannot be 13 total digits.")
 
         return cls(message_type=FullMessageType.PASSTHROUGH_COMMAND, body=body)
+
+    @classmethod
+    def passthrough_uart_keycode_numeric_body_and_mac(cls, secret_key):
+        """Use a given secret key to generate a passthrough keycode
+        :param secret_key: secret key used to generate UART security key
+        :type secret_key: byte
+        """
+        mac = compute_passthrough_uart_keycode_numeric_body_and_mac(secret_key)
+        return FactoryFullMessage.passthrough_command(
```

Comment:
```suggestion
        return cls.passthrough_command(
```

This method already shares a class with the classmethod that we are trying to call, so we can just use `cls` here, which is automatically passed to `passthrough_uart_keycode_numeric_body_and_mac` method as the first argument.